client/src/components/products/EditProduct.js

```diff
@@ -0,0 +1,500 @@
+            type="number"
+            name="price"
+            margin="normal"
+            variant="outlined"
+            value={this.state.price}
+            onChange={this.handleChange}
+            InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">EUR</InputAdornment>
+              )
+            }}
+          />
+          {/* Company */}
+          <TextField
+            id="outlined-company-input"
+            label="Company Name"
+            className={classes.textField}
+            type="text"
+            name="company"
+            placeholder="IchiBizz Inc"
+            autoComplete="company"
+            margin="normal"
+            variant="outlined"
+            value={this.state.company}
+            onChange={this.handleChange}
+          />
+          {/* Location (read-only) this is populated from the map*/}
+          <TextField
+            disabled
+            id="outlined-location"
+            label="Location"
+            className={classes.textField}
+            type="text"
+            name="address"
+            placeholder="e.g. Berlin"
+            margin="normal"
+            variant="outlined"
+            value={this.state.location.address}
+            onChange={this.getMapData}
+          />
+          {/* Availability */}
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Grid container justify="space-around">
+              <KeyboardDatePicker
+                disableToolbar
+                name="availability"
+                variant="inline"
+                format="MM/dd/yyyy"
+                margin="normal"
+                id="date-picker-inline"
+                label="Availability Date"
+                value={this.state.availability}
+                onChange={this.handleDateChange}
+                KeyboardButtonProps={{
+                  "aria-label": "Change Date"
+                }}
+              />
+            </Grid>
+          </MuiPickersUtilsProvider>
+          {/* Warranty Until */}
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Grid container justify="space-around">
+              <KeyboardDatePicker
+                disableToolbar
+                name="warrantyUntil"
+                variant="inline"
+                format="MM/dd/yyyy"
+                margin="normal"
+                id="date-picker-inline"
+                label="Warranty until"
+                value={this.state.warrantyUntil}
+                onChange={this.handleDateChangeWarrantyUntil}
+                KeyboardButtonProps={{
+                  "aria-label": "Change Date"
+                }}
+              />
+            </Grid>
+          </MuiPickersUtilsProvider>
+          {/* Condition */}
+          <FormControl
+            required
+            component="fieldset"
+            className={classes.formControl}
+          >
+            <FormLabel component="legend">Condition</FormLabel>
```

Comment:
since of styling  ( i  am still struggling material UI.... ) 
  it will be changed <h3>Condition</h3>